### PR TITLE
Feature Freeze In Effect

### PR DESCRIPTION
**Pull request by Fox-McCloud**
All PRs submitted after this will not be considered for merging if they are a feature, sprite, or balance PR----or a refactor PR that impacts balance. Submitting a PR that falls within this category will result in the PR being instantaneously closed, regardless of how good the PR is. Current projected end date for the freeze is January 1, 2017.

Performance, fix, and refactor PRs will all be allowed and considered for merging.

### CODE REVIEW

**Comment by tristan1333** [comment]
BOY he doin' it

**Comment by DarkPyrolord** [comment]
Is it just me, or did it suddenly get really cold here?

**Comment by TheDZD** [comment]
No ice puns.
0/10

**Comment by PJB3005** [comment]
@TheDZD you need to chill.

**Comment by KasparoVy** [comment]
Lol, wow. Guess I get to take a vacation for a bit

**Comment by monster860** [comment]
Needs more labels

**Comment by Wheatley29** [comment]
Is there an actual point to this?

**Comment by Dinarzad** [comment]
To encourage fixing some of the 200+ bugs listed atm, one would assume.

**Comment by Iamgoofball** [comment]
Because shutting down development for 4+ months is a great idea.

**Comment by Plotron** [comment]
We also have some large scale feature PRs that are being worked on currently.

**Comment by Taaketa** [comment]
There are some 200 bugs at the moment and at lewst 3 HUGE PRs that are coming that will bring their own bugs.

This freeze is plain to me as to why it's enacted.

**Comment by tristan1333** [comment]
http://nanotrasen.se/phpBB3/viewtopic.php?f=48&t=8413 pretty sure that's where the idea came from for future reference  to everyone

**Comment by LittleBigKid2000** [comment]
It would probably be a good idea to go through all the issues and close the fixed/irrelevant ones.

**Comment by AndriiYukhymchak** [comment]
Please run thru issues that we have and close ones that are not reproducable or requested to be closed... there are a lot of em

**Comment by Markolie** [comment]
What's the current state/plans with regards to the freeze?

**Comment by Iamgoofball** [comment]
i like how a 3 month freeze did nothing but stagnate the codebase for 3 months

**Comment by orelbon** [comment]
I suggest getting more coders if you want bugs getting fixed, not freezing development.

**Comment by fludd12** [comment]
I decided to be annoying and prune some of the issue log by repeatedly asking if things needed to still be issues.

Sue me.

**Comment by monster860** [comment]
You are a god now. @fludd12

File: code/modules/reagents/newchem/pyro.dm [hunk 1]
```diff
@@ -468,7 +468,7 @@
 /datum/reagent/cryostylane/on_mob_life(mob/living/M) //TODO: code freezing into an ice cube
 	if(M.reagents.has_reagent("oxygen"))
 		M.reagents.remove_reagent("oxygen", 1)
-		M.bodytemperature -= 30
+		M.bodytemperature -= 1000000
```
Comment (jughu) [inline code/modules/reagents/newchem/pyro.dm]: `M.bodytemperature -= 21` is the fix

**Comment by fludd12** [comment]
Truly, my nigh-omnipotence is proven by my ability to spam "is this still a bug"

**Comment by Spacemanspark** [comment]
Death to feature freeze

**Comment by Norgad** [comment]
2016-12-31 13_00_03-greenshot
👀👀👀👀👀👀👀👀👀👀👀

**Comment by Fox-McCloud** [comment]
img

**Comment by Norgad** [comment]
Somthing somthing time-zones

**Comment by jughu** [comment]
We better have our meme prs ready when new year hits for the US :smirk:
Would be a shame if we lost precious time

**Comment by F-OS** [comment]
12:59 MST, THE WAIT IS MADDENING!

**Comment by Norgad** [comment]
Are we there yet?

**Comment by Crazylemon64** [comment]
soon :tm:

**Comment by Signmark** [comment]
oh god CONFLICTS

**Comment by fludd12** [comment]
The freeze is down!
Release the gitKraken!

**Comment by BiberDark** [comment]
image

Summer is coming

**Comment by tigercat2000** [comment]
freeze isn't dead till the pr is closed 😏

**Comment by BiberDark** [comment]
..still open ?

**Comment by Crazylemon64** [comment]
http://media.riffsy.com/images/7ff5c42d79dfb1bff7e022baed670114/raw

**Comment by Crazylemon64** [comment]
brace for the storm

**Pull request closed without merging**